developer-docs-site/docs/cli-tools/aptos-cli-tool/install-aptos-cli.md

```diff
@@ -14,58 +14,250 @@ If you want to use the Move Prover, then, [install the Move Prover dependencies]
+2. In the latest release section, you will see the zip files with the filename of the format: `aptos-cli-<version>-<platform>`. These are the platform-specific pre-compiled binaries of the CLI. Download the zip file for your platform.
+3. Unzip the downloaded file. This will extract the `aptos` CLI binary file into your default downloads folder. 
+4. Move this extracted `aptos` binary file into your preferred local folder. 
+
+   :::tip Upgrading? Remember to look in the default download folder
+   When you update the CLI binary with the latest version, note that the newer version binary will be downloaded to your default Downloads folder. Remember to move this newer version binary from the Downloads folder to `~/bin/aptos` folder (overwriting the older version).
+   :::
+
+5. Make this `~/bin/aptos` as an executable by running this command:
+    - `chmod +x ~/bin/aptos`.
+6. Type `~/bin/aptos help` to read help instructions.
+7. Add `~/bin` to your path in your `.bashrc` or `.zshrc` file for future use.
+
+</details>
+
+<details>
+<summary>Windows 10, 11 and Windows Server 2022+</summary>
+
+### Windows 10, 11 and Windows Server 2022+
+
+:::tip
+These instructions have been tested on Windows 11 and Windows Server 2022. Windows support is new and some features may be not complete. Open [Github issues](https://github.com/aptos-labs/aptos-core/issues) for bugs.
+:::
+
+1. Go to the [Aptos CLI release page](https://github.com/aptos-labs/aptos-core/releases?q=cli&expanded=true).
+2. In the latest release section, you will see the zip files with the filename of the format: `aptos-cli-<version>-<platform>`. These are the platform-specific pre-compiled binaries of the CLI. Download the zip file for your platform.
+3. Unzip the downloaded file. This will extract the `aptos` CLI binary file into your default downloads folder. For example, on Windows it is the `\Users\user\Downloads` folder.
+4. Move this extracted `aptos` binary file into your preferred local folder.
+   :::tip Upgrading? Remember to look in the default download folder
+   When you update the CLI binary with the latest version, note that the newer version binary will be downloaded to your default Downloads folder. Remember to move this newer version binary from the Downloads folder to your preferred location.
+   :::
+5. Open a powershell terminal via the windows start menu
+6. In the powershell terminal, you can get help instructions by running the command with help.  For example ` .\Downloads\aptos-cli-0.3.5-Windows-x86_64\aptos.exe help` to read help instructions.
+
+</details>
 
 ## (Optional) Install the dependencies of Move Prover
 
 If you want to use the Move Prover, install the dependencies by following the below steps:
 
-:::tip Windows is not supported
-The Move Prover is not supported on Windows.
+:::tip
+Currently Windows is not supported for the prover.
+:::
+
+<details>
+<summary>Prover MacOS installation</summary>
```

Comment:
To make this consistent, probably just MacOS, similar for linux below

Reply:
It is consistent. The Linux below is with `###`, like the `MacOS` here. Best seen in the preview: https://deploy-preview-4543--aptos-developer-docs.netlify.app/